website/docs/r/hpc_cache.html.markdown

```diff
@@ -93,17 +101,67 @@ An `access_rule` block contains the following:
 * `root_squash_enabled` - (Optional) Whether to enable [root squash](https://docs.microsoft.com/en-us/azure/hpc-cache/access-policies#root-squash)? Defaults to `false`.
 
 * `anonymous_uid` - (Optional) The anonymous UID used when `root_squash_enabled` is `true`.
-  
+ 
 * `anonymous_gid` - (Optional) The anonymous GID used when `root_squash_enabled` is `true`.
 
 ---
 
+A `bind` block contains the following:
+
+* `dn` - (Required) The Bind Distinguished Name (DN) identity to be used in the secure LDAP connection.
+ 
+* `password` - (Required) The Bind password to be used in the secure LDAP connection.
+
+---
+
 A `default_access_policy` block contains the following:
 
 * `access_rule` - (Required) One to three `access_rule` blocks as defined above.
 
 ---
 
+A `directory_ad` block contains the following:
+
+* `primary_dns` - (Required) The primary DNS IP address used to resolve the Active Directory domain controller's FQDN.
+
+* `domain_name` - (Required) The fully qualified domain name of the Active Directory domain controller.
+ 
+* `cache_net_bios_name` - (Required) The NetBIOS name to assign to the HPC Cache when it joins the Active Directory domain as a server.
+
+* `domain_net_bios_name` - (Required) The Active Directory domain's NetBIOS name.
+
+* `username` - (Required) The username of the Active Directory domain administrator.
+ 
+* `password` - (Required) The password of the Active Directory domain administrator.
+
+* `secondary_dns` - (Optional) The secondary DNS IP address used to resolve the Active Directory domain controller's FQDN.
+
+---
+
+A `directory_flat_file` block contains the following:
+
+* `group_file_uri` - (Required) The URI of the file containing group information (`/etc/group` file format in Unix-like OS).
+
+* `passwd_file_uri` - (Required) The URI of the file containing user information (`/etc/passwd` file format in Unix-like OS).
+
+---
+
+A `directory_ldap` block contains the following:
+
+* `server` - (Required) The FQDN or IP address of the LDAP server.
+
+* `base_dn` - (Required) The base distinguished name (DN) for the LDAP domain.
+
+* `conn_encrypted` - (Optional) Whether the LDAP connection should be encrypted? Defaults to `false`.
+
+* `certificate_validation_uri` - (Optional) The URI of the CA certificate to validate the LDAP secure connection.
+
+* `download_certificate` - (Optional) Whether the certificate should be automatically downloaded. This can be set to `true` only when `certificate_validation_uri` is provided. Defaults to `false`.
```

Comment:
```suggestion
* `download_certificate_automatically` - (Optional) Whether the certificate should be automatically downloaded. This can be set to `true` only when `certificate_validation_uri` is provided. Defaults to `false`.
```